src/cmd/cmd.cpp

```diff
@@ -311,7 +311,7 @@ int main(int argc, char **argv)
     qputenv("OPENSSL_CONF", opensslConf.toLocal8Bit());
 #endif
 
-    qsrand(QTime::currentTime().msec() * QCoreApplication::applicationPid());
+    qsrand(QDateTime::currentMSecsSinceEpoch() * QCoreApplication::applicationPid());
```

Comment:
could this overflow? both calls return (signed) qint64  
  Epoch should be ~40 bits and i don't know how many bits is PID on windows for example.
could be worth casting to unsiged.

I would suggest   qHash(QDateTime::currentMSecsSinceEpoch()) ^ qhash(QCoreApplication::applicationPid())